sources/packages/forms/src/form-definitions/partnerinformationandincomeappeal.json

```diff
@@ -180,25 +180,25 @@
         {
           "label": "Enter your spouse/common-law partner's reported total income from line 15000 of their {{data.calculatedTaxYear}} income tax return. If they did not file a {{data.calculatedTaxYear}} income tax return, enter their total income from all sources both inside and outside of Canada.",
           "prefix": "$",
-          "customClass": "font-weight-bold",
+          "applyMaskOn": "change",
           "mask": false,
-          "spellcheck": true,
           "tableView": false,
-          "currency": "CAD",
+          "delimiter": true,
+          "requireDecimal": false,
           "inputFormat": "plain",
+          "truncateMultipleSpaces": false,
           "validate": {
-            "required": true
+            "required": true,
+            "custom": "valid = (input >= 0 && input < data.maxIncome) ? true : 'The number you have entered is outside the expected range.';"
```

Comment:
This appeal has a different hidden variable `maxMoneyValue` unlike the applications which have `maxIncome` and also `maxMoneyValue` seems to be more generic to be used for a currency input.

Reply:
Updated.